acceptance.bats

```diff
@@ -290,5 +290,11 @@
 @test "The number of tests run is accurate" {
   run ./conftest test -p examples/kubernetes/policy examples/kubernetes/service.yaml --no-color
   [ "$status" -eq 0 ]
-  [ "${lines[1]}" = "5 tests, 4 passed, 1 warning, 0 failures" ]
+  [ "${lines[1]}" = "5 tests, 4 passed, 1 warning, 0 failures, 0 exceptions" ]
+}
+
+@test "Exceptions reported correctly" {
+  run ./conftest test -p examples/exceptions/policy examples/exceptions/deployments.yaml --no-color
+  [ "$status" -eq 1 ]
+  [ "${lines[2]}" = "2 tests, 0 passed, 0 warnings, 1 failure, 1 exception" ]
```

Comment:
Shouldn't this check for `exceptions` instead of `exception`?

Reply:
Since it is just 1 exception, it shouldn't be pluralized, right? I basically just did the same pattern as the others on that output line.
Or am I misunderstanding?

Reply:
No it makes sense. I missed the fact that for singular form we had a different output.